### PR TITLE
Add travis builds

### PR DESCRIPTION
**Pull request by f1ori**
Soon I won't have access to our internal CI anymore. As I'll sure miss the green lights for this project, I'd propose to build this on travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.3"
+  - "2.7"
```
Comment (xhochy) [inline .travis.yml]: Why not also 3.4?

**Comment by lorenzhs** [comment]
:+1:

**Comment by leonhandreke** [comment]
Maybe we can add one of those nice badges to our README? Not sure how that works.

**Comment by lorenzhs** [comment]
You can copy the code for them somewhere on the travis page and just paste it into the readme iirc

**Pull request merged**